css/styles.css

```diff
@@ -1,3 +1,132 @@
 html {
-  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+}
+
+body {
+  font-family: "Courier New", Courier, monospace;
+  background-color: black;
+  color: antiquewhite;
+}
+
+header {
+  text-align: center;
+}
+
+.main {
+  width: 80%;
+  margin: auto;
+}
+
+.main-container {
+  border: solid rgb(209, 209, 207);
+  background-color: rgb(117, 116, 116);
+  width: 100%;
+  max-height: 300px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.main-container ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+li {
+  display: flex;
+  justify-content: center;
+  list-style-type: none;
+  color: #fff;
+  padding: 0.2em;
+  padding-left: 0;
+  padding-top: 0.5em;
+  margin-left: -2.3em;
+}
+
+li .title-span {
+  display: grid;
+  grid-template-columns: auto auto;
+  word-break: break-all;
+  width: 100%;
+  border: solid;
+  border-radius: 20% 10%;
+  background-color: blueviolet;
+  height: 2em;
+  margin-left: 0;
+  padding: 0.1em;
+  padding-bottom: 0.3em;
+}
+
+.remove-btn {
+  color: antiquewhite;
+  background-color: rgba(216, 72, 72, 0.2);
+  margin-left: -2.4em;
+  height: 2em;
+}
+
+.remove-btn:hover {
+  color: rgb(219, 40, 40);
+}
+
+.remove-btn:active {
+  background-color: rgba(209, 25, 25, 1);
+  color: antiquewhite;
+}
+
+form {
+  display: flex;
```

Comment:
- [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

Reply:
Thanks. We have implemented the changes in the next feature